### ISSUE TITLE
`num_chains` documentation implies filenames are different in sequential and parallel cases

### ISSUE THREAD

**Issue by WardBrian** [stan-dev/docs#615]
#### Summary:
The [Multi Chain Sampling](https://mc-stan.org/docs/cmdstan-guide/mcmc-intro.html#multi-chain-sampling) section includes the following sentence

> If the model was not compiled with `STAN_THREADS=true`, the above command will run 4 chains sequentially and will produce the sample in `output_1.csv`, `output_2.csv`, `output_3.csv`, `output_4.csv`.

This sentence is confusing because the output file names are the same in the sequential and parallel case, but as written it implies that the output filenames are also part of the conditional statement being made.

This prompted [this forum question](https://discourse.mc-stan.org/t/running-chains-on-multiple-cores/30212)

**Comment by bob-carpenter** [comment]
I don't think this actually implies they are different, but I can see how someone will be confused.  

All I would do to clarify is add a parenthetical, "(the same files as when run in parallel)".  

The content should be the same either way, too, because the seeds are distinct and the same in the parallel and sequential case.

**Comment by WardBrian** [comment]
This was cleaned up in https://github.com/stan-dev/docs/pull/752
